Makefile

```diff
@@ -60,4 +60,4 @@ test: test-all
 	./scripts/check-coverage.sh $(PWD)
 
 clean:
-	rm -rf debug release test-all bench test/*.o deps/url-cpp/{debug,release}
```

Comment:
For my own edification, what were the braces doing that was causing problems?

Reply:
Make wasn't performing the brace expansion so the directories didn't get deleted. I believe it's because make uses `/bin/sh` which does not support brace expansion.

Reply:
😞